test/fixtures/es-modules/invalid.cjs

```diff
@@ -0,0 +1 @@
+import "invalid";
```

Comment:
Can you make sure to also include another test for `export`? Maybe a few `export` variations as well.